public/video-ui/styles/layout/_display.scss

```diff
@@ -0,0 +1,33 @@
+.video-display {
+  padding: 10px;
+  display: grid;
+
+  grid-gap: 10px;
+  grid-template-columns: 1fr 1fr;
+  grid-template-areas:
+    "select select"
+    "metadata workflow"
+    "metadata preview"
+    "metadata usage"
+    "metadata .";
```

Comment:
using `.` as a trick (creates an empty spacing element) to avoid excessive spacing